helmfile/values/falco.yaml.gotmpl

```diff
@@ -182,7 +182,6 @@ customRules:
         and not mcafee_writing_cma_d
         and not avinetworks_supervisor_writing_ssh
         and not multipath_writing_conf
-        and not calico_node
```

Comment:
How come this is needed?

Reply:
https://github.com/elastisys/compliantkubernetes-apps/pull/948/files

To fix a bug of this. Right now, Falco will crash in the main branch.. at least when I did this.